qt/python/mantidqtinterfaces/mantidqtinterfaces/SampleTransmissionCalculator/stc_view.py

```diff
@@ -19,13 +19,21 @@ def __init__(self, parent=None):
         fig = Figure()
         self.axes = fig.add_subplot(111)
         self.plot_frame = FigureCanvas(fig)
```

Comment:
When resizing to a small size, the axis labels can run off the edge of the plot area and become hidden. Giving the figure a tight layout is the simplest way to fix this.

```suggestion
        fig = Figure()
        fig.set_layout_engine(layout="tight")
        self.axes = fig.add_subplot(111)
        self.plot_frame = FigureCanvas(fig)
```